app/src/components/ExportPane.js

```diff
@@ -3,73 +3,56 @@ import React, { useMemo } from 'react';
 import s from './ExportPane.module.css';
 import cs from 'clsx';
 
-
 import displayNames from '../displayNames';
 
-function ResultRow({
-  result
-}) {
-
+function ResultRow({ result }) {
```

Comment:
Are there any changes in this file besides the addition of the key or is it just all formatting?

Reply:
Nope. Nothing besides those changes